### PR TITLE
Documentation/dask client example

### PR DESCRIPTION
**Pull request by benjamc**
Add example how to setup a dask client for a SLURM cluster. #998

### CODE REVIEW

File: examples/1_basics/7_parallelization_cluster.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+The facade works best on a numerical hyperparameter configuration space and should not
+be applied to problems with large evaluation budgets (up to 1000 evaluations).
+"""
+
+import numpy as np
+from ConfigSpace import Configuration, ConfigurationSpace, Float
+from dask.distributed import Client
+from dask_jobqueue import SLURMCluster
+from matplotlib import pyplot as plt
+
```
Comment (sarah-segel) [inline examples/1_basics/7_parallelization_cluster.py:20]: Could take out matplotlib / RunHistory

**Review by sarah-segel — APPROVED** [review]

File: examples/1_basics/7_parallelization_cluster.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        processes=1,
+        log_directory="tmp/smac_dask_slurm",
+    )
+    cluster.scale(jobs=n_workers)
+
+    # Dask will create n_workers jobs on the cluster which stay open.
+    # Then, SMAC/Dask will schedule individual runs on the workers like on your local machine.
+    client = Client(
+        address=cluster,
+    )
```
Comment (eddiebergman) [inline examples/1_basics/7_parallelization_cluster.py:94]: You can also do `cluster.get_client()` which is a bit easier and requires one less import :)

Comment (benjamc) [reply examples/1_basics/7_parallelization_cluster.py:94]: I wasn't aware of that, added a note :)

File: examples/1_basics/7_parallelization_cluster.py (hunk 1 above)
Comment (benjamc) [reply examples/1_basics/7_parallelization_cluster.py:20]: Done

**Pull request merged**